### PR TITLE
Enabling selecting filtering of diff output

### LINKED ISSUE #11: Allow suppression of actual diff by `kind`
First of all, super helpful tool that we use quite often!  Thanks for your time.

We would love to integrate `helm-diff` a bit tighter in our CI/CD pipeline but have a bit of an issue where we don't want to expose the actual `diff` of a credential to the output. 

It would be extremely handy to be able to blacklist a set of `kind` resources that would only show a `CONTENT CHANGED` type message if anything internal had changed instead of actually displaying the contents.

### PR DESCRIPTION
**Pull request by jrnt30**
Fixes #11 

- Allows easy suppression of secret content diff via the `-q` or `--suppress-secrets` flags 
- Allows additional customization of suppression via the `--suppress` flag

### CODE REVIEW

**Comment by databus23** [comment]
Thanks for the PR. I was on vacation in the last weeks and I"m still in the progress of catching up. I will try to look into this tomorrow. Thanks for your patience.

**Comment by jrnt30** [comment]
Not a problem at all, hope you enjoyed your time off!

File: main.go [hunk 1]
```diff
@@ -59,8 +62,10 @@ func main() {
 
 	f := cmd.Flags()
 	f.BoolP("version", "v", false, "show version")
+	f.BoolP("suppress-secrets", "q", false,  "suppress secrets in the output")
 	f.VarP(&diff.valueFiles, "values", "f", "specify values in a YAML file (can specify multiple)")
 	f.StringArrayVar(&diff.values, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
+	f.StringArrayVar(&diff.suppressedKinds, "suppress", []string{}, "suppress secrets in the output")
```
Comment (databus23) [inline main.go]: The description is not accurate I think. It should mention You can specify kinds and that you have to specify it multiple times to exclude more then one.

**Comment by databus23** [comment]
Thanks!

**Pull request merged**